engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -53,19 +57,64 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
   val tag = self.path.name
   startWith(WorkflowExecutionPendingState, WorkflowExecutionActorData())
 
-  def startJob(call: Call) = {
-    // TODO: Support indexes and retries:
-    val jobKey = BackendJobDescriptorKey(call, None, 1)
-    val jobDescriptor: BackendJobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, Map.empty)
-    val executionActor = backendForExecution(jobDescriptor, workflowDescriptor.backendAssignments(call))
-    executionActor ! ExecuteJobCommand
+  /** PBE: the return value of WorkflowExecutionActorState is just temporary.
+    *      This should probably return a Try[BackendJobDescriptor], Unit, Boolean,
+    *      Try[ActorRef], or something to indicate if the job was started
+    *      successfully.  Or, if it can fail to start, some indication of why it
+    *      failed to start
+    */
+  private def startJob(jobKey: BackendJobDescriptorKey,
+                       inputs: Map[FullyQualifiedName, WdlValue],
+                       configDescriptor: BackendConfigurationDescriptor,
+                       factory: BackendLifecycleActorFactory): WorkflowExecutionActorState = {
+    val jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, inputs)
+    val jobExecutionActor = context.actorOf(
+      factory.jobExecutionActorProps(
+        jobDescriptor,
+        BackendConfigurationDescriptor(configDescriptor.backendConfig, CromwellConfiguration.config)
```

Comment:
Why swap out the passed in `configDescriptor.globalConfig`, instead of passing through the `configDescriptor` as is to the `factory.jobExecutionActorProps`?